services/frontend/src/components/SkillsBadges/SkillBadges.js

```diff
@@ -0,0 +1,43 @@
+import React, {useMemo} from "react";
+import classNames from "classnames";
+import "./SkillBadges.scss";
+
+const SkillBadges = ({displayCount, skills}) => {
+    const mappedSkillBadges = useMemo(() => skills.slice(0, displayCount).map((skill) => {
+        return (
+            <SkillBadge
+                key={skill.name}
+                name={skill.name}
+                isHighlySkilled={skill.isHighlySkilled}
+            />
+        );
+    }), [skills, displayCount]);
+
+    return (
+        <>
+            {mappedSkillBadges}
+        </>
+    );
+};
+
+const SkillBadge = ({name, isHighlySkilled = false}) => (
+    <TextBadge
+        className="skill-badge"
+        text={name}
+        isHighlighted={isHighlySkilled}
+    />
+);
+
+const TextBadge = ({className, text, isHighlighted = false}) => (
```

Comment:
Now the `TextBadge` component is duplicated in two places: `SkillBadges.js` and `ProjectCard.js`. I suggest refactoring it out into its own component file.

Reply:
Sounds good. 

Moved it into `components/TextBadge`